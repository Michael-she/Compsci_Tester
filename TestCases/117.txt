l c 4 3
l c 5 3
d c 4 5
d c 5 5
s 1 0 3
s 1 9 3
#
4 3 d
1 3 d
4 5 r
4 6 r
5 3 u
6 3 u
