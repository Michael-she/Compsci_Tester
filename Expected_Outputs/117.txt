pygame 2.5.2 (SDL 2.28.3, Python 3.8.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  | c|  | C|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  | c|  | C|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  | c|  | C|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  | C|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |13|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |13|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  | c|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  | c|  | C|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  | C|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  | c|  | C|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  | C|46|46|  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  | c|  | C|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  | C|
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |63|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |63|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  | c|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  | C|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  | C|
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  | C|  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  |  |  | C|
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  | s|  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
Light wins!
